src/odemis/gui/conf/util.py

```diff
@@ -101,36 +101,52 @@ def binning_1d_from_2d(comp, va, conf):
         logging.warning("Got a binning not of length 2: %s, will try anyway", cur_val)
 
     try:
-        nbpx_full = comp.shape[0] * comp.shape[1]  # res at binning 1
-        choices = {cur_val[0]}
-        minbin = max(1, max(va.range[0]))  # Force minimum binning 1 (for scanners with scale < 1)
-        maxbin = min(va.range[1])
-        if "range_1d" in conf:
-            conf_rng = conf["range_1d"]
-            minbin = max(conf_rng[0], minbin)
-            maxbin = min(conf_rng[1], maxbin)
-
-        # add up to 5 binnings
-        b = int(math.ceil(minbin))  # in most cases, that's 1
-        for _ in range(6):
-            # does it still make sense to add such a small binning?
-            if b > cur_val[0]:
-                nbpx = nbpx_full / (b ** 2)
-                if nbpx < MIN_RES: # too small resolution
-                    break
-                if len(choices) >= 5: # too many choices
+        if hasattr(va, "choices"):
+            # Pass all available choices which are same in both dimensions
+            choices = sorted(x for x, y in va.choices if x==y)
```

Comment:
No. Here this function only shows one value for X and Y, so we only want to pass the values where X==Y.
Note that it's fine for the FASTEM. The *resolutions* in xt_client can have a different ratio, but all the scale are identical in X & Y: the pixels are the same size in both dimensions. Conveniently, for now, all resolutions map to different scales, which is convenient as there is a mapping scale <-> resolution. In theory, it could be different, but we could just pick the resolution that corresponds to the biggest FoV.